Sources/SQL/Core/Data.swift

```diff
@@ -31,14 +31,19 @@ public struct ValueConversionError: ErrorProtocol {
 public enum SQLData {
     case Text(String)
     case Binary(Data)
+    case RawSQL(String)
 }
 
-public protocol SQLDataConvertible {
+public protocol SQLDataRepresentable {
```

Comment:
Can you give some rationale for this change. Throughout Zewo, we have been following a `-Convertible` naming convention when a type can be _converted_ both to and from a common type (in this case, `SQLData`).